src/wix/WixToolset.Core.WindowsInstaller/CommandLine/WindowsInstallerCommand.cs

```diff
@@ -45,6 +45,10 @@ public bool TryParseArgument(ICommandLineParser parser, string argument)
             {
                 switch (argument.ToLowerInvariant())
                 {
+                    case "decompile":
+                        this.Subcommand = new DecompilerSubcommand(this.ServiceProvider);
+                        return true;
+
```

Comment:
Please update the help text when adding new commands. I believe some are missing in Burn, too.

Reply:
I have two work items where I will go through all the command lines and scrub them. They are all way out of date.

Reply:
👎